Tests/TestUtilities/SQPropertiesFileReader.cs

```diff
@@ -30,22 +31,12 @@ namespace TestUtilities
     /// </summary>
     public class SQPropertiesFileReader
     {
-        // NB: this expression only works for single-line values
-        // Regular expression pattern: we're looking for matches that:
-        // * start at the beginning of a line
-        // * start with a character or number
-        // * are in the form [key]=[value],
-        // * where [key] can  
-        //   - starts with an alpanumeric character.
-        //   - can be followed by any number of alphanumeric characters or .
-        //   - whitespace is not allowed
-        // * [value] can contain anything
-        public const string KeyValueSettingPattern = @"^(?<key>\w[\w\d\.-]*)=(?<value>[^\r\n]+)";
+
 
         /// <summary>
         /// Mapping of property names to values
         /// </summary>
-        private IDictionary<string, string> properties;
+        private JavaProperties properties;
```

Comment:
I copy pasted a big code to load Java properties (this is complex, due to multi-line syntax, escaping, ...). So in the end I prefer to not touch this code. Note that JavaProperties internally rely on a HashTable so I think this is pretty similar to a IDictionary